.github/PULL_REQUEST_TEMPLATE/add_repo.md

```diff
@@ -0,0 +1,27 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+* Consider creating this PR as draft: https://github.com/kubevirt/kubevirt/blob/main/CONTRIBUTING.md#consider-opening-your-pull-request-as-draft
+-->
+
+**What this PR does / why we need it**:
+
+This PR adds the repository `repo_name` to the KubeVirt organization.
+
+_Checklist:_
+
+- [ ] repo is a good fit for the org
+- [ ] maintainers are ok with adding the repo
```

Comment:
Let's be specific, we refer to these maintainers https://github.com/kubevirt/community/blob/main/MAINTAINERS.md right?
Please link to it.

I#d also rephrase it to: "Maintainers who sponsor this repository: "

sponsor or support. But we need accountability from the maintainers.